index.bs

```diff
@@ -1508,11 +1514,11 @@ these steps:
 
   1. Perform the <a spec=html>activation notification</a> steps in |global|'s [=Window/browsing context=].
```

Comment:
I was thinking this should probably be after the new requesting permission steps, to match the implementation. But that probably means that the permission request steps in this algorithm should probably duplicate some of the permission request algorithm, rather than relying on calling the abstraction for it, since otherwise spec-wise it should fail if there is no current user activation at this point in the algorithm.

So the logic here then probably becomes:
 - create a FileSystemPermissionDescriptor
 - call "request permission to use" with the descriptor
 - call the "boolean permission query algorithm" to get the result of the request.

Reply:
This has been updated in the latest patch. Please confirm it's what we want :)